peer/brontide.go

```diff
@@ -75,6 +75,11 @@ var (
 	// either the Brontide doesn't know of it, or the channel in question
 	// is pending.
 	ErrChannelNotFound = fmt.Errorf("channel not found")
+
+	// pongBuf is a slice we'll reuse instead of allocating memory on the
+	// heap. Since only reads will occur and no writes, there is no need
+	// for any synchronization primitives.
+	pongBuf = make([]byte, lnwire.MaxPongBytes)
```

Comment:
Hmm, sort of weird to allocate this much memory on the package level...think I'll instead make this a new required config option.

Reply:
Ok re-worked to get rid of this. We use a new struct level variable, that's initialize by the server and retained across all goroutines.